src/browser.js

```diff
@@ -32,7 +32,7 @@ const NOT_SUPPORTED = 'The browser target is not supported: %target%';
  * Launches the specified browser with the given URL.
  * Based on https://github.com/domenic/opener
  * @param {{target: ?string, url: ?string, dataDir: ?string}} opts - parameters:
- *   target - the target browser - ie, chrome, safari, opera, firefox or chromium
+ *   target - the target browser - ie, edge, chrome, safari, opera, firefox or chromium
```

Comment:
```suggestion
 *   target - the target browser - ie, edge,edgechromium, chrome, safari, opera, firefox or chromium
```

For the sake of completeness

Reply:
I think `ie` in this case was refering to "Internet Explorer" and not "In Example"

Reply:
Whoops